check_pgactivity

```diff
@@ -7255,18 +7255,27 @@ DB_LOOP: foreach my $stat (@rs) {
 
 =item B<uptime> (8.1+)
 
-Returns time since postmaster start (from 8.1) and configuration reload (from 8.4).
+Returns time since postmaster start ("uptime", from 8.1),
+since configuration reload (from 8.4),
+and since shared memory initialization (from 10).
+
+Please note that the uptime is unaffected when the postmaster resets
+all its children (for example after a kill -9 on a process or a failure).
+
+From 10+, the 'time since shared memory init' aims at detecting this situation:
+in fact we use the age of the oldest non-client child process (usually checkpointer,
+writer or startup). This needs pg_monitor access to read pg_stat_activity.
 
 Critical and Warning thresholds are optional. If both are set, Critical is
-raised when the postmaster uptime is less than the critical threshold, and
-Warning is raised when the configuration uptime is less than the warning thereshold.
+raised when the postmaster uptime or the time since shared memory initialization
+is less than the critical threshold.
+Warning is raised when the time since configuration reload is less than the warning threshold.
 If only a warning or critical threshold is given, it will be used for both cases.
 Obviously these alerts will disappear from themselves once enough time has passed.
 
-Perfdata contain the duration since postmaster start and duration since
-configuration reload.
+Perfdata contain the three values, or NULL if not available for this version.
```

Comment:
Perfdata should not include data not supported on the hosts. Even if set to NULL.
```suggestion
Perfdata contains each values depending of the postgresql version and their availability.
```

Reply:
Corrected in my  branch